Xamarin.Forms.Core/BindableObject.cs

```diff
@@ -321,12 +324,14 @@ internal void SetValue(BindableProperty property, object value, bool fromStyle)
 			SetValue(property, value, fromStyle, true);
 		}
 
-		internal void SetValueCore(BindablePropertyKey propertyKey, object value, SetValueFlags attributes = SetValueFlags.None)
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public void SetValueCore(BindablePropertyKey propertyKey, object value, SetValueFlags attributes = SetValueFlags.None)
 		{
 			SetValueCore(propertyKey.BindableProperty, value, attributes, SetValuePrivateFlags.None);
 		}
 
-		internal void SetValueCore(BindableProperty property, object value, SetValueFlags attributes = SetValueFlags.None)
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public void SetValueCore(BindableProperty property, object value, SetValueFlags attributes = SetValueFlags.None)
```

Comment:
As the API is browser attributed it doesn't constitute part of our supported public surface area and as such could be changed back to internal at a later state. I have no objection to doing SetvalueCore -> SetValueFromRenderer scrub but just not as part of this PR; conversion to SetValueFromRenderer is not necessary for IVT removal and as a non-trivial change could potentially break something.